content/trial_getting_started/examples/kubernetes/_index.md

```diff
@@ -0,0 +1,99 @@
++++
+title = "Level 7: Multi-Cluster Kubernetes Orchestration"
+description = "Learn how to design your service topology such that it is abstracted from the infrastructure layer and can be deployed on AWS using a single, flexible template."
+weight = 70
+alwaysopen = false
++++
+
+This example demonstrates defining an environment independent Kubernetes workload and deploying it to development and production clusters.  The example runs on Amazon Web Services (AWS) and so requires and account and API credentials.  It available on [github](https://github.com/cloudify-community/eaas-example).
+
+The infrastructure deployment consists of [`ServiceComponent`s](../../../working_with/service_composition) representing:
+
+ * network
+ * database
+ * object store
+ * kubernetes
+
+The implementation of these components is based on the environment type, which is supplied
+as an input to the deployment.  For example, if a development environment is selected, <a href="http://minio.io">Minio</a> is used as the object store and <a href="https://minikube.sigs.k8s.io/docs/">Minikube</a> as the Kubernetes provider.  If a production environment is selected, <a href="https://aws.amazon.com/s3">S3</a> and <a href="https://aws.amazon.com/eks">EKS</a> is used for each, respectively.
```

Comment:
You might check with marketing if they want to use a special tracking URL to see who clicks out of our website.